tensorflow/python/eager/core_test.py

```diff
@@ -965,11 +965,6 @@ def testOperationWithNoInputsRunsOnDevice(self):
     execute(
         b'Add', 1, inputs=[x, y], attrs=('T', x.dtype.as_datatype_enum))
 
-  def testInvalidDevice(self):
-    with self.assertRaises(ValueError):
-      with context.device('pu:0'):
-        _ = constant_op.constant(1)
-
```

Comment:
This test is checking whether the device name is a valid device name(statically defined in python/framework/device_spec.py), however, since PluggableDevice's device type is dynamically registered from plugin, this check need to be removed. 
```
# EPU represents for TPU embedding for now. Subject to change in future.
_VALID_DEVICE_TYPES = frozenset({"CPU", "GPU", "TPU", "CUSTOM", "EPU"})
```